documentation.md

```diff
@@ -0,0 +1,131 @@
+# strong-error-handler documentation
+
+This middleware is intended to be used in both development (debug) and production environments.
+
+## Production mode
+
+- Error responses output do not contain sensitive details or  information.
+- Only `safeFields` are generated in the output.
```

Comment:
`safeFields` options was not implemented yet, see https://github.com/strongloop/strong-error-handler/issues/7 

```
For 5xx errors, the output contains only the status code and the status name from the HTTP specification.

For 4xx errors, the output contains the full error message (`error.message`) and the contents of the `details` property (`error.details`). The latter is typically used by `ValidationError` to provide machine-readable details about the validation problems.
```